python/Ganga/GPIDev/Lib/File/LCGSEFile.py

```diff
@@ -286,18 +286,13 @@ def getWNInjectedScript(self, outputFiles, indent, patternsToZip, postProcessLoc
 
         return script
 
-    def get(self):
+    def copyTo(self, targetPath):
```

Comment:
If the file has already been downloaded then this should just copy it from the localDir. This probably applies to all remote file types. As such, putting a default logic to the effect in the base class might make sense.